src/otx/algo/modules/conv_module.py

```diff
@@ -248,26 +243,26 @@ def init_weights(self) -> None:
         # Note: For PyTorch's conv layers, they will be overwritten by our
         #    initialization implementation using default ``kaiming_init``.
         if not hasattr(self.conv, "init_weights"):
-            if self.with_activation and self.act_cfg["type"] == "LeakyReLU":
+            if self.with_activation and self.act_cfg["type"] == "LeakyReLU":  # type: ignore[index]
                 nonlinearity = "leaky_relu"
-                a = self.act_cfg.get("negative_slope", 0.01)
+                a = self.act_cfg.get("negative_slope", 0.01)  # type: ignore[union-attr]
             else:
                 nonlinearity = "relu"
                 a = 0
             kaiming_init(self.conv, a=a, nonlinearity=nonlinearity)
         if self.with_norm:
             constant_init(self.norm_layer, 1, bias=0)
 
-    def forward(self, x: torch.Tensor, activate: bool = True, norm: bool = True) -> torch.Tensor:
+    def forward(self, x: Tensor, activate: bool = True, norm: bool = True) -> Tensor:
         """Forward pass of the ConvModule.
 
         Args:
-            x (torch.Tensor): Input tensor.
+            x (Tensor): Input tensor.
             activate (bool, optional): Whether to apply activation. Defaults to True.
             norm (bool, optional): Whether to apply normalization. Defaults to True.
 
         Returns:
-            torch.Tensor: Output tensor.
+            Tensor: Output tensor.
         """
         layer_index = 0
         while layer_index < len(self.order):
```

Comment:
Yes, it is not an essential feature, but `DepthwiseSeparableConvModule` uses `Torch` instead of `torch.Tensor`.
And in OTX `Torch` seems to be used more than `torch.Tensor`.
So I just tried to match the format used within the file and follow this trend.

Reply:
Yes, I don't have any objections about using Torch for typing. My comment was about "self.order". This parameter set the order of the components (conv, norm, act). I wonder do we change the order somewhere and it is useful feature or not. Could we remove "order" functionality from the ConvModule?

Reply:
Sorry for the misunderstanding.
It seems not to be used anywhere, I'll remove it and check integration tests.
ec353de